### ISSUE TITLE
Prozessing files bigger than 10MB lead to: thread 'main' has overflowed its stack

### ISSUE THREAD

**Issue by JoKalliauer** [RazrFalcon/svgcleaner#117]
Processing a file above 10MB with following comand:
>$ svgcleaner [Stations_Vélhop.svg](https://commons.wikimedia.org/wiki/File:Stations_V%C3%A9lhop.svg) output.svg

lead to an error:
`
thread 'main' has overflowed its stack
`

**Comment by RazrFalcon** [comment]
It's actually the svgdom bug and I have no idea how to fix it. Will try to find a solution.

**Comment by RazrFalcon** [comment]
Also, it crashes only in the debug mode.

**Comment by JoKalliauer** [comment]
The strange thing is that svgcleaner reduced the file size, and it has all closing tags, it ends with `</g></svg>`, and it is valid file and it looks like the original file (30MB), therefore I'm not shure if it really crashes.

**Comment by RazrFalcon** [comment]
The problem is that it crashes during the DOM destructing, after file being processed.

**Comment by RazrFalcon** [comment]
Basically, I have to rewrite [svgdom](https://github.com/RazrFalcon/libsvgdom) which is will not happen anytime soon.

**Comment by JoKalliauer** [comment]
You maybe already know it:
I noticed that not the filesize it is the number of tags.
I get problems with a file which has 10.000 path-tags but is less than 300kB, but I don't get any problems with a file using a single path-tag, but bigger than 10MB.

I tried Processing:
[https://commons.wikimedia.org/wiki/File:1_42_polytope_7-cube.svg](https://commons.wikimedia.org/wiki/File:1_42_polytope_7-cube.svg)
once before merging 480.000paths (using [svgo](https://github.com/svg/svgo)) and once afterwards, afterwards I didn't had any problems.

Actually a function `merging path` would also be a nice feature. (Merging more than 10.000Path is very slow in [svgo](https://github.com/svg/svgo), and svgo is generally more buggy than your script.)

But if it is too complex, it's maybe not worth it.
